Build/libHttpClient.141.Android.Java/app/src/main/java/com/xbox/httpClient/HttpClientResponse.java

```diff
@@ -0,0 +1,61 @@
+
+package com.xbox.httpclient;
+
+import android.util.Log;
+
+import java.io.IOException;
+import okhttp3.Response;
+
+class HttpClientResponse
+{
+    private Response okHttpResponse;
```

Comment:
>okHttpResponse [](start = 21, length = 14)

why is it called okHttpResponse if it can be unsuccessful?

Reply:
That's the worst name for a namespace I've ever seen...

---
In reply to: [190003661](https://github.com/Microsoft/libHttpClient/pull/165#discussion_r190003661) [](ancestors = 190003661,189760525)